libraries/plugins/market_history/market_history_plugin.cpp

```diff
@@ -408,7 +408,7 @@ void market_history_plugin::plugin_set_program_options(
    )
 {
    cli.add_options()
-         ("bucket-size", boost::program_options::value<string>()->default_value("[60,300,900,1800,3600,14400,86400]"),
+         ("bucket-size", boost::program_options::value<string>()->default_value("[15,60,300,3600,86400]"),
```

Comment:
i saw a comment from @HarukaMa about this but it seems it was deleted. he was pointing at this line as unintended and he is right, this should remain `[60,300,900,1800,3600,14400,86400]`

Reply:
I was confused by f213db3 and thought this was intended though...

Reply:
hmm, i am confused now as well, the commit you mention was a reverse of a mistake i made pushing code directly to master but if we merge this as it is it will reverse again, i am sure that is unintended. the change should be removed making sure `[60,300,900,1800,3600,14400,86400]` are the final settings.

Reply:
This seems to be a github issue, i rebased to develop and force push. Don't see this changeset locally in git.

Reply:
I can see it locally after pulling from origin.

Reply:
fixed